context/app/static/js/components/detailPage/Attribution/Attribution.jsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
-import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+
 import { useFlaskDataContext } from 'js/components/Contexts';
 import { DetailPageSection } from 'js/components/detailPage/style';
 import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
+import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+
 import { FlexPaper } from './style';
 import SectionItem from '../SectionItem';
 
 function Attribution() {
   const {
-    entity: { group_name, created_by_user_displayname, created_by_user_email },
+    entity: { group_name, created_by_user_displayname, created_by_user_email, entity_type },
   } = useFlaskDataContext();
 
   return (
     <DetailPageSection id="attribution">
-      <SectionHeader>Attribution</SectionHeader>
+      <SectionHeader iconTooltipText={`Information about the group registering this ${entity_type}`}>
```

Comment:
I believe `entity_type` should be in lowercase. You can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase on `entity_type.

Reply:
Good catch .